docs/app.md

```diff
@@ -352,18 +352,19 @@ Download the filesystem of an app within a project to your local machine
 ```
 USAGE
   $ mw app download [INSTALLATION-ID] --target <value> [-q] [--ssh-user <value>] [--ssh-identity-file <value>]
-    [--exclude <value>...] [--dry-run] [--delete]
+    [--exclude <value>...] [--dry-run] [--delete] [--sub-directory <value>]
 
 ARGUMENTS
   INSTALLATION-ID  ID or short ID of an app installation; this argument is optional if a default app installation is set
                    in the context.
 
 FLAGS
-  -q, --quiet               suppress process output and only display a machine-readable summary.
-      --delete              delete local files that are not present on the server
-      --dry-run             do not actually download the app installation
-      --exclude=<value>...  [default: ] exclude files matching the given pattern
-      --target=<value>      (required) target directory to download the app installation to
+  -q, --quiet                  suppress process output and only display a machine-readable summary.
+      --delete                 delete local files that are not present on the server
+      --dry-run                do not actually download the app installation
+      --exclude=<value>...     [default: ] exclude files matching the given pattern
+      --sub-directory=<value>  specify a sub-directory within the app installation to download
+      --target=<value>         (required) target directory to download the app installation to
```

Comment:
For a Moment I was confused why I can not set the `sub-directory` directly as part of `target` … then I noticed the difference:
* `target` is the local path on my machine
* `sub-directory` is a relative path on the remote machine

Maybe the flag should be called `remote-sub-dir` to better reflect this?

Reply:
Good catch. :+1:  I like `remote-sub-dir` for its better clarity; will rename...